woof-code/rootfs-petbuilds/compat-chroot/usr/sbin/compat-chroot

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh -e
+
+BINDS="/dev /proc /sys /tmp /run /root /home/spot /usr/share/fonts /etc/fonts /usr/share/fontconfig /bin/busybox /bin/ash /usr/sbin/run-as-spot /usr/sbin/setup-spot /etc/hosts /etc/resolv.conf"
```

Comment:
TODO: in Puppy, /var/run/dbus is a symlink to /run/dbus, but in Debian, /var/run is a symlink to /run, creating a circular dependency that makes the D-Bus socket inaccessible